commander/src/commands/network-identifier.ts

```diff
@@ -21,16 +21,16 @@ import { flags as commonFlags } from '../utils/flags';
 
 export default class NetworkIdentifierCommand extends BaseCommand {
 	static description = `
-  Creates Network identifier for the given nethash and community identifier.
+  Creates Network identifier for the given genesis payload hash and community identifier.
 	`;
 
 	static examples = [
-		'network-identifier --nethash=da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
+		'network-identifier da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
```

Comment:
missing args `--genesisPayloadHash`

Reply:
@shuse2 said I had too remove that. Shu?

Reply:
Yes, actually this example was wrong, the nethash was not flag, it's arg

Reply:
@ManuGowda check answer and resolve this and related please